Bump WordPress "tested up to" version 6.7

jeffpaul opened the pull request:
<!--
Filling out this template is required.  Any PR that does not include enough information to be reviewed may be closed at a maintainers' discretion.  All new code requires documentation and tests to ensure against regressions.
-->

### Description of the Change
@severine-pozzo confirmed in https://github.com/10up/publisher-media-kit/issues/211#issuecomment-2465290681 that things passed in WordPress 6.7 testing, so this PR bumps to that version and drags forward the min version to 6.5.

<!-- Enter any applicable Issue number(s) here that will be closed/resolved by this PR. -->
Closes #211.

### How to test the Change
<!-- Please provide steps on how to test or validate that the change in this PR works as described. -->

### Changelog Entry
<!--
Please include a summary for this PR, noting whether this is something being Added / Changed / Deprecated / Removed / Fixed / or Security related.  You can replace the sample entries after this comment block with the single changelog entry line for this PR. -->
> Changed - Bump WordPress "tested up to" version 6.7
> Changed - Bump WordPress minimum supported version to 6.5.

### Credits
<!-- Please list any and all contributors on this PR so that they can be added to this projects CREDITS.md file. -->
Props @severine-pozzo @jeffpaul.

### Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you are unsure about any of these, please ask for clarification.  We are here to help! -->
- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [ ] I have updated the documentation accordingly.
- [ ] I have added [Critical Flows, Test Cases, and/or End-to-End Tests](https://10up.github.io/Open-Source-Best-Practices/testing/) to cover my change.
- [ ] All new and existing tests pass.

jeffpaul committed:
bump WordPress "tested up to" version 6.7

Co-Authored-By: severine-pozzo <107071518+severine-pozzo@users.noreply.github.com>

jeffpaul commented:
@severine-pozzo looks like automated e2e tests failed on WP `trunk` (aka 6.7).  Can you check and see if there's something perhaps not working on this version?  cc: @faisal-alvi

jeffpaul commented:
@faisal-alvi can you help check out that failure?

severine-pozzo commented:
I will be retesting this morning @Jeff Paul ***@***.***>

On Mon, Nov 11, 2024 at 11:48 PM Jeffrey Paul ***@***.***>
wrote:

> @faisal-alvi <https://github.com/faisal-alvi> can you help check out that
> failure?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/10up/publisher-media-kit/pull/212#issuecomment-2469534666>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZQ4QHQ2TJT6W5XXBE4XCFD2AF3B7AVCNFSM6AAAAABROHUD5WVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDINRZGUZTINRWGY>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

*Severine (Seve) Pozzo*
Lead Quality Assurance Specialist · 10up, Inc. <https://10up.com/> · @10up
<https://twitter.com/10up>


*email address: ***@***.*** ***@***.***>time
zone:  Atlantic Standard Time (AST) UTC -4 **Please note that I am off on
Friday afternoon*

*pronouns: she, her*

faisal-alvi committed:
fix e2e tests

faisal-alvi committed:
fix eslint errors,
/home/runner/work/publisher-media-kit/publisher-media-kit/tests/cypress/integration/pmk-block-patterns.test.js
Error:    1:1   error  'describe' is not defined                                                                                                                                                                                                                                                                                                                               no-undef
Error:    2:1   error  Replace `····` with `↹`                                                                                                                                                                                                                                                                                                                                 prettier/prettier
Error:    2:5   error  'it' is not defined                                                                                                                                                                                                                                                                                                                                     no-undef
Error:    3:1   error  Replace `········` with `↹↹`                                                                                                                                                                                                                                                                                                                            prettier/prettier
Error:    3:9   error  'cy' is not defined                                                                                                                                                                                                                                                                                                                                     no-undef
Error:    4:1   error  Replace `········` with `↹↹`                                                                                                                                                                                                                                                                                                                            prettier/prettier
Error:    4:9   error  'cy' is not defined                                                                                                                                                                                                                                                                                                                                     no-undef
Error:    5:1   error  Replace `········cy.get('#post-title-0,·h1.editor-post-title__input').click(·{·force:·true·}·)` with `↹↹cy.get('#post-title-0,·h1.editor-post-title__input')⏎↹↹↹.click({·force:·true·})⏎↹↹↹`                                                                                                                                                            prettier/prettier
Error:    5:9   error  'cy' is not defined                                                                                                                                                                                                                                                                                                                                     no-undef
Error:    6:1   error  Replace `········cy.get('.edit-post-header-toolbar__inserter-toggle,·.editor-document-tools__inserter-toggle'` with `↹↹cy.get(⏎↹↹↹'.edit-post-header-toolbar__inserter-toggle,·.editor-document-tools__inserter-toggle',⏎↹↹`                                                                                                                            prettier/prettier
Error:    6:9   error  'cy' is not defined                                                                                                                                                                                                                                                                                                                                     no-undef
Error:    7:1   error  Replace `········cy.get('.components-tab-panel__tabs·button,·.block-editor-inserter__tabs·button,·.block-editor-tabbed-sidebar__tablist·button').contains(·'Patterns'·)` with `↹↹cy.get(⏎↹↹↹'.components-tab-panel__tabs·button,·.block-editor-inserter__tabs·button,·.block-editor-tabbed-sidebar__tablist·button',⏎↹↹)⏎↹↹↹.contains('Patterns')⏎↹↹↹`  prettier/prettier
Error:    7:9   error  'cy' is not defined                                                                                                                                                                                                                                                                                                                                     no-undef
Error:    9:1   error  Replace `········` with `↹↹`                                                                                                                                                                                                                                                                                                                            prettier/prettier
Error:   10:1   error  Replace `········` with `↹↹`                                                                                                                                                                                                                                                                                                                            prettier/prettier
Error:   10:9   error  'cy' is not defined                                                                                                                                                                                                                                                                                                                                     no-undef
Error:   11:1   error  Replace `············` with `↹↹↹`                                                                                                                                                                                                                                                                                                                       prettier/prettier
Error:   12:1   error  Replace `················` with `↹↹↹↹`                                                                                                                                                                                                                                                                                                                  prettier/prettier
Error:   12:17  error  'cy' is not defined                                                                                                                                                                                                                                                                                                                                     no-undef
Error:   13:1   error  Replace `····················` with `↹↹↹↹↹`                                                                                                                                                                                                                                                                                                             prettier/prettier
Error:   14:1   error  Replace `················` with `↹↹↹↹`                                                                                                                                                                                                                                                                                                                  prettier/prettier
Error:   15:1   error  Replace `············}·else·if·($body.find(·'[aria-label="Publisher·Media·Kit"]'·` with `↹↹↹}·else·if·($body.find('[aria-label="Publisher·Media·Kit"]'`                                                                                                                                                                                                 prettier/prettier
Error:   16:1   error  Replace `················` with `↹↹↹↹`                                                                                                                                                                                                                                                                                                                  prettier/prettier
Error:   16:17  error  'cy' is not defined                                                                                                                                                                                                                                                                                                                                     no-undef
Error:   17:1   error  Replace `············` with `↹↹↹`                                                                                                                                                                                                                                                                                                                       prettier/prettier
Error:   19:1   error  Replace `············` with `↹↹↹`                                                                                                                                                                                                                                                                                                                       prettier/prettier
Error:   20:1   error  Replace `············` with `↹↹↹`                                                                                                                                                                                                                                                                                                                       prettier/prettier
Error:   20:13  error  'cy' is not defined                                                                                                                                                                                                                                                                                                                                     no-undef
Error:   21:1   error  Replace `········` with `↹↹`                                                                                                                                                                                                                                                                                                                            prettier/prettier
Error:   22:1   error  Replace `····` with `↹`                                                                                                                                                                                                                                                                                                                                 prettier/prettier

faisal-alvi committed:
a few more eslint fixes

faisal-alvi committed:
update the command in lint workflow

faisal-alvi committed:
trying the direct command

faisal-alvi committed:
ignore tests from eslint check

faisal-alvi committed:
revert package name changes

faisal-alvi committed:
fix eslint issues in existing tests
